modules/ztp-adding-new-content-to-gitops-ztp.adoc

```diff
@@ -3,72 +3,146 @@
+    #Elasticsearch Operator
+    - fileName: elasticsearch/ElasticsearchNS.yaml <1>
+      remediationAction: inform 
+      policyName: "group-dev-elasticsearch-ns" 
+    - fileName: elasticsearch/ElasticsearchOperatorGroup.yaml
+      remediationAction: inform 
+      policyName: "group-dev-elasticsearch-operator-group" 
+    #Bell Custom Resources
+    - fileName: bell/apiserver-config.yaml
+      remediationAction: inform 
+      policyName: "group-dev-apiserver-config" 
+    - fileName: bell/disable-nic-lldp.yaml
+      remediationAction: inform 
+      policyName: "group-dev-disable-nic-lldp" 
 ----
+<1> Set `fileName` to include the custom CR subfolder from the `/source-crs` parent.
 
-. Open a terminal at the `ztp-update/` folder and rebuild the container:
-+
-[source,terminal,subs="attributes+"]
-----
-$ podman build -t ztp-site-generate-rhel8-custom:v{product-version}-custom-1
-----
+. Commit the `PolicyGenTemplate` change in Git, and then push to the Git repository being monitored by the GitOps ZTP Argo CD policies application.
 
-. Push the built container image to your disconnected registry, for example:
+. Update the `ClusterGroupUpgrade` CR to include the changed `PolicyGenTemplate` and save it as `cgu-test.yaml`.
 +
-[source,terminal,subs="attributes+"]
+[source,yaml]
 ----
-$ podman push localhost/ztp-site-generate-rhel8-custom:v{product-version}-custom-1 registry.example.com:5000/ztp-site-generate-rhel8-custom:v{product-version}-custom-1
+apiVersion: ran.openshift.io/v1alpha1
+kind: ClusterGroupUpgrade
+metadata:
+  name: custom-source-cr
+  namespace: ztp-clusters
+spec:
+  managedPolicies:
+    - group-dev-config-policy
+  enable: true
+  clusters:
+  - cluster1
+  remediationStrategy:
+    maxConcurrency: 2
+    timeout: 240
 ----
 
-. Patch the Argo CD instance on the hub cluster to point to the newly built container image:
+. Apply the updated `ClusterGroupUpgrade` CR by running the following command:
+[source, terminal]
 +
-[source,terminal,subs="attributes+"]
 ----
-$ oc patch -n openshift-gitops argocd openshift-gitops --type=json -p '[{"op": "replace", "path":"/spec/repo/initContainers/0/image", "value": "registry.example.com:5000/ztp-site-generate-rhel8-custom:v{product-version}-custom-1"} ]'
+$ oc apply -f cgu-test.yaml
 ----
-+
-When the Argo CD instance is patched, the `openshift-gitops-repo-server` pod automatically restarts.
 
 .Verification
 
-. Verify that the new `openshift-gitops-repo-server` pod has completed initialization and that the previous repo pod is terminated:
+. Check that the updates have succeeded by running the following command:
 +
-[source,terminal]
+[source, terminal]
 ----
-$ oc get pods -n openshift-gitops | grep openshift-gitops-repo-server
+$ oc  get cgu -A
 ----
 +
 .Example output
 +
-[source,terminal]
+[source, terminal]
 ----
-openshift-gitops-server-7df86f9774-db682          1/1     Running   	     1          28s
+NAMESPACE     NAME               AGE   STATE        DETAILS
```

Comment:
Nit: the example output is mis-aligned in the HTML

Reply:
Aidan is correct. The sample output is misaligned. It's not a big enough issue to prevent a merge, but please write up a small Jira to correct the typo. Thanks!

Reply:
Corrected this in new PR https://github.com/openshift/openshift-docs/pull/64949